module2/homework/nwdNww/nwdNww.hpp

```diff
@@ -1,11 +1,13 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+   if (rhs!=0)
+        return NWD (rhs,lhs%rhs);
+        return std::abs(lhs); 
```

Comment:
Here also it would be nice to format the code with proper spacing. For example `if (rhs != 0)`.

And here also curly brackets would be helpful. At first I thought that both return belongs to if statement. Especially because both lines with `return` have the same indentation level.